Tests/MapKitTests/MKPolylineTests.swift

```diff
@@ -36,6 +44,20 @@ final class MKPolylineTests: XCTestCase {
         }
     }
 
+    func testInitWithLocations() {
+        var refCoordinates = locations.map {
+            return CLLocationCoordinate2D(latitude: $0.coordinate.latitude, longitude: $0.coordinate.longitude)
+        }
+
+        let polyline = MKPolyline(locations: locations)
+        let polyline2 = MKPolyline(coordinates: &refCoordinates, count: refCoordinates.count)
+
+        for (coordinate1, coordinate2) in zip(polyline.coordinates, polyline2.coordinates) {
```

Comment:
Why not just use `zip(polyline.coordinates, refCoordinates)`, because the way it is being done right now we testing against the same implementation, so the result will always be the same...

Reply:
i basically copied and pasted from the test above.
`    func testInitWithCoordinates() {
        var refCoordinates = coordinates

        let polyline = MKPolyline(coordinates: coordinates)
        let polyline2 = MKPolyline(coordinates: &refCoordinates, count: refCoordinates.count)

        for (coordinate1, coordinate2) in zip(polyline.coordinates, polyline2.coordinates) {
            XCTAssertEqual(coordinate1.latitude, coordinate2.latitude, accuracy: 0.000000001)
            XCTAssertEqual(coordinate1.longitude, coordinate2.longitude, accuracy: 0.000000001)
        }
    }`

Reply:
I agree with @LucianoPAlmeida regarding re-using the implementation. I don't like that it was done above either. Just compare `polyline.coordinates` with `coordinates`.